lib/spree_virtual_gift_card/factories.rb

```diff
@@ -12,6 +12,11 @@
 
     factory :redeemable_virtual_gift_card do
       association :purchaser, factory: :user
+      inventory_unit do |gift_card|
+        gift_card.line_item.inventory_units.first ||
```

Comment:
i see. what if there are multiple gift cards? would they be associated to the same inventory unit? then if both get canceled that could cause trouble